connectors/sources/sharepoint.py

```diff
@@ -219,17 +224,18 @@ async def get_content(
 
         source_file_name = ""
 
-        async with NamedTemporaryFile(mode="wb", delete=False) as async_buffer:
-            async for response in self._api_call(
-                url_name=ATTACHMENT,
-                host_url=self.host_url,
-                value=site_url,
-                file_relative_url=file_relative_url,
-            ):
-                async for data in response.content.iter_chunked(CHUNK_SIZE):
-                    await async_buffer.write(data)
+        async for response in self._api_call(
+            url_name=ATTACHMENT,
+            host_url=self.host_url,
+            value=site_url,
+            file_relative_url=file_relative_url,
+        ):
+            async with self.limiter:
+                async with NamedTemporaryFile(mode="wb", delete=False) as async_buffer:
+                    async for data in response.content.iter_chunked(CHUNK_SIZE):
+                        await async_buffer.write(data)
```

Comment:
Should `self.limiter` be here? Does `iter_chunked` do request per chunk?

Reply:
Yeah, `iter_chunked` do request per chunk.